platform/app/src/appInit.js

```diff
@@ -67,6 +68,7 @@ async function appInit(appConfigOrFunc, defaultExtensions, defaultModes) {
     UserAuthenticationService.REGISTRATION,
     PanelService.REGISTRATION,
     StateSyncService.REGISTRATION,
+    ColorbarService.REGISTRATION,
```

Comment:
I haven't looked at the service itself, but I think this is not a "Core" service, so move it to cornerstone extension (similar to ToolGroupService and CornerstoneViewportService which are cornerstone services)

Reply:
Done